main.php

```diff
@@ -117,6 +117,8 @@
 require_once('core/startup/ErrorControlChain.php');
 require_once('core/startup/ParameterConfirmationToken.php');
 
+ini_set('display_errors', 0);
```

Comment:
The old code disabled the error within SilverStripe's error trapper, so that errors were not displayed twice. I believe it also incorporated the  `error_reporting()` check that @camspiers mentioned below.

Putting `display_errors` at the top here is awkward—it was placed here (rather than, for example, in the code where the handlers are registered) because of ErrorControlChain; what we probably need to do is introduce new functionality into the error handler. This may mean subclassing the monolog error handler, or contributing to it.

I had hoped we could do this in the `HTTPOutputHandler` class, but this is called *after* the built-in handler, as best I can see. I'll re-test, though.